libs/repositories/src/utils/fetch.ts

```diff
@@ -0,0 +1,14 @@
+export async function throwIfUnsuccessful(
```

Comment:
File name doesn't match export

Reply:
`fetch` was just meant to be having a bunch of utils for fetch. For now only one. Anyways, I can rename the file if you prefer

Reply:
I would prefer, since looking at it seems like a mistake.
Or explain it in a comment/docstring like you just told me.

Reply:
no worries, I did the change when I wrote this comment